Мерж

Tech3434 opened the pull request:
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You'll find a README and example file in .\html\changelogs\ for further instructions.

You can also find a template for adding your changelog directly to the PR description here: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation
-->

AffectedArc07 commented:
Please disable mapdiffbot and icondiffbot from this repo. You are causing rate limits and service failures

AffectedArc07 commented:
Пожалуйста, отключите mapdiffbot и icondiffbot из этого репо. Вы вызываете ограничения скорости и сбои в обслуживании

AffectedArc07 commented:
Can this be closed already?

emmanuelbassil committed:
Tweaks reagent_container feeding

Spookerton committed:
Merge pull request #35659 from l3lb0t/doublestack-2

Fixes doublestack pistol mag rubber and practice sprites

Bay12Bot committed:
Automatic changelog generation for PR #35659 [ci skip]

dependabot[bot] committed:
Bump actions/checkout from 6.0.1 to 6.0.2

Bumps [actions/checkout](https://github.com/actions/checkout) from 6.0.1 to 6.0.2.
- [Release notes](https://github.com/actions/checkout/releases)
- [Commits](https://github.com/actions/checkout/compare/v6.0.1...v6.0.2)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-version: 6.0.2
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Spookerton committed:
Merge pull request #35660 from Baystation12/dependabot/github_actions/actions/checkout-6.0.2

Bump actions/checkout from 6.0.1 to 6.0.2

Bay12Bot committed:
Automatic changelog generation [ci skip]

HeyBanditoz committed:
C4 buffs

* C4 now destroys the girder left behind from destroying a wall
* C4 explosive power has been buffed

MuckerMayhem committed:
Merge pull request #35662 from HeyBanditoz/c5

C4 buffs

Bay12Bot committed:
Automatic changelog generation for PR #35662 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

MuckerMayhem committed:
research matrix progress persistence

Bay12Bot committed:
Automatic changelog generation [ci skip]

CakeQ committed:
Merge pull request #35657 from MuckerMayhem/research-matrix-persistence

Make research matrix progress persistent between rounds

MuckerMayhem committed:
fix vines stacking when spreading vertically

Spookerton committed:
base produce types are instantiable by item path

Added a PRODUCE_SEED define that simplifies creating grown items (fruits,
etc) in spawning, WillContain, etc. by giving them a path to reference.

MuckerMayhem committed:
replace airlock circuits with braces in storage

MuckerMayhem committed:
fix resort solars

MuckerMayhem committed:
working resort map for events

CakeQ committed:
Merge pull request #35669 from MuckerMayhem/working-resort-map

Working resort map for events

Bay12Bot committed:
Automatic changelog generation for PR #35669 [ci skip]

dependabot[bot] committed:
Bump actions/cache from 5.0.2 to 5.0.3

Bumps [actions/cache](https://github.com/actions/cache) from 5.0.2 to 5.0.3.
- [Release notes](https://github.com/actions/cache/releases)
- [Changelog](https://github.com/actions/cache/blob/main/RELEASES.md)
- [Commits](https://github.com/actions/cache/compare/8b402f58fbc84540c8b491a91e594a4576fec3d7...cdf6c1fa76f9f475f3d7449005a359c84ca0f306)

---
updated-dependencies:
- dependency-name: actions/cache
  dependency-version: 5.0.3
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Spookerton committed:
Merge pull request #35668 from MuckerMayhem/resort-solar-fix

fix resort solars

Bay12Bot committed:
Automatic changelog generation for PR #35668 [ci skip]

Spookerton committed:
Merge pull request #35670 from Baystation12/dependabot/github_actions/actions/cache-5.0.3

Bump actions/cache from 5.0.2 to 5.0.3

Spookerton committed:
Merge pull request #35665 from MuckerMayhem/vine-fix

fix vines stacking when spreading vertically

Bay12Bot committed:
Automatic changelog generation for PR #35665 [ci skip]

Spookerton committed:
Merge pull request #35667 from MuckerMayhem/airlock-bracezx

replace airlock circuits with braces in hard storage

Bay12Bot committed:
Automatic changelog generation for PR #35667 [ci skip]

LordNest committed:
[FIX] Remove Hypospray master object from ERT ship

MuckerMayhem committed:
Merge pull request #35663 from emmanuelbassil/PillIngest

Tweaks reagent_container feeding

Bay12Bot committed:
Automatic changelog generation for PR #35663 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

HeyBanditoz committed:
Merge pull request #35666 from Spookerton/spkrtn/bay/cng/productive

base produce types are instantiable by item path

MuckerMayhem committed:
move command messages to singleton

MuckerMayhem committed:
add a fridge with random fruits/veggies

MuckerMayhem committed:
Merge pull request #35671 from LordNest/LN-ERT-Hypo

[FIX] Remove Hypospray master object from ERT ship

Bay12Bot committed:
Automatic changelog generation [ci skip]

Spookerton committed:
Merge pull request #35664 from MuckerMayhem/fridge-veggies

add a fridge with random fruits/veggies to the kitchen

Bay12Bot committed:
Automatic changelog generation for PR #35664 [ci skip]

LordNest committed:
[ICONS] Jumpsuit, Baton and plastic spoon

Bay12Bot committed:
Automatic changelog generation [ci skip]

CakeQ committed:
Merge pull request #35672 from MuckerMayhem/command-msg-singletoning

move command messages to singletons

Bay12Bot committed:
Automatic changelog generation for PR #35672 [ci skip]

Spookerton committed:
Merge pull request #35673 from LordNest/LN-03026

[ICONS] Jumpsuit, Baton and plastic spoon

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

LordNest committed:
[FIX] Robots telekinesis removal and wired connection sprite fix

Bay12Bot committed:
Automatic changelog generation [ci skip]

HeyBanditoz committed:
Merge pull request #35675 from LordNest/LN-060226

[FIX] Robots telekinesis removal and wired connection sprite fix

MuckerMayhem committed:
tag out sticky notes for engineering

MuckerMayhem committed:
add sound when drinking from some glasses

Bay12Bot committed:
Automatic changelog generation [ci skip]

CakeQ committed:
Merge pull request #35677 from MuckerMayhem/drank

add sound when drinking from some glasses

Bay12Bot committed:
Automatic changelog generation for PR #35677 [ci skip]

MuckerMayhem committed:
fix zcopy issues when loading maps at runtime

MuckerMayhem committed:
Merge pull request #35676 from MuckerMayhem/tag-out

tag out sticky notes for engineering

Bay12Bot committed:
Automatic changelog generation for PR #35676 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

SierraKomodo committed:
Merge pull request #35678 from MuckerMayhem/runtime-mapload-fix

fix zcopy issues when loading maps at runtime

Bay12Bot committed:
Automatic changelog generation for PR #35678 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Spookerton committed:
add maptext font preference

applies it to all maptext except status displays
status displays in particular are multiline- for user determined
fonts this causes presentation issues I'm not interested in solving

l3lb0t committed:
Adds a neutral doublestack pistol

HeyBanditoz committed:
Make it easier to use mindbreaker in food for imprinting

* Move all base drug type + mindbreaker effects to metabolism instead of blood
* All drugs have 1.0 bioavailability

LordNest committed:
[UI] Assorted browser restyling for Fake NanoUI

MuckerMayhem committed:
Merge pull request #35681 from l3lb0t/doublestack-3

Adds a New Doublestack Pistol

Bay12Bot committed:
Automatic changelog generation for PR #35681 [ci skip]

MuckerMayhem committed:
Merge pull request #35683 from HeyBanditoz/mindbreaker_food_fix

Make it easier to use mindbreaker in food for imprinting

Bay12Bot committed:
Automatic changelog generation for PR #35683 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

HeyBanditoz committed:
Fix sensor program not working on telescreens

HeyBanditoz committed:
Merge pull request #35684 from LordNest/LN-12-02-26-Fake

[UI] Assorted browser restyling for Fake NanoUI

HeyBanditoz committed:
Merge pull request #35686 from HeyBanditoz/sensor_telescreen_fix

Fix sensor program not working on telescreens

Bay12Bot committed:
Automatic changelog generation for PR #35684 [ci skip]

Bay12Bot committed:
Automatic changelog generation for PR #35686 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Spookerton committed:
clients with roles can observe before the game is ready

Bay12Bot committed:
Automatic changelog generation [ci skip]

LordNest committed:
[ICONS] Xeno stomaches

HeyBanditoz committed:
Merge pull request #35690 from LordNest/LN-dev-staging-02

[ICONS] Xeno stomaches

Bay12Bot committed:
Automatic changelog generation for PR #35690 [ci skip]

HeyBanditoz committed:
Merge pull request #35689 from Spookerton/spkrtn/bay/cng/how-the-sausage-is-made

clients with roles can observe before the game is ready

Bay12Bot committed:
Automatic changelog generation for PR #35689 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

HeyBanditoz committed:
Consolidate CL file creation and compile steps

Beforehand, creating CL files (from PR descriptions) and compiling them
were separate jobs where the latter ran on a daily schedule, or manually
invoked by a dev.

This commit consolidates those two steps into one, so the changelog HTML
and .all_changelog.yml file are updated immediately after a PR is merged
into dev. In other words, no more waiting or having to manually run
the compile changlogs job to update the changelog.

The compilation step will be skipped if no changelog was generated from
a PR.

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

SierraKomodo committed:
Examining charging stations shows occupant

SierraKomodo committed:
Drag+Drop people into charging stations

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

MuckerMayhem committed:
add timestamps to tcom logs, fix corrupted logs

Bay12Bot committed:
Automatic changelog generation [ci skip]

MuckerMayhem committed:
Merge pull request #35693 from SierraKomodo/tweak/charging-station-qol

Charging Station QOLs

Bay12Bot committed:
Automatic changelog generation for PR #35693 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

HeyBanditoz committed:
Use better font + scrolling text for status displays

Also includes sticky additive lighting fix

HeyBanditoz committed:
Fix "Notify Wiki On Merge" action failing on merge

MuckerMayhem committed:
Merge pull request #35694 from MuckerMayhem/tcomms-log-things

add timestamps to tcom logs, fix corrupted logs

Bay12Bot committed:
Automatic changelog generation for PR #35694 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

HeyBanditoz committed:
Fix fax machine GC failures

HeyBanditoz committed:
Merge pull request #35699 from HeyBanditoz/fax_machine_destroy_fix

Fix fax machine GC failures

HeyBanditoz committed:
Merge pull request #35696 from HeyBanditoz/improved_status_displays

Use better font + scrolling text for status displays

Bay12Bot committed:
Automatic changelog generation for PR #35699 [ci skip]

Bay12Bot committed:
Automatic changelog generation for PR #35696 [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Bay12Bot committed:
Automatic changelog generation [ci skip]

Spookerton committed:
Merge pull request #35691 from HeyBanditoz/consolidated_changelog

Consolidate CL file creation and compile steps

Spookerton committed:
Revert "roller beds now have wheels"

This reverts commit c96e71a6bed6a597e2fc1b584b24198d5325573c.

CakeQ committed:
Merge pull request #35700 from Spookerton/spkrtn/bay/cng/revert-35593

Revert "roller beds now have wheels"

HeyBanditoz committed:
Fix CL jobs pushing to protected branch

Use bot token instead of GHA token

Spookerton committed:
Merge pull request #35701 from HeyBanditoz/consolidated_changelog_pt2

Fix CL jobs pushing to protected branch

Bay12Bot committed:
Automatic changelog generation for PR #35701 [ci skip]

dependabot[bot] committed:
Bump dorny/paths-filter from 3.0.2 to 4.0.0

Bumps [dorny/paths-filter](https://github.com/dorny/paths-filter) from 3.0.2 to 4.0.0.
- [Release notes](https://github.com/dorny/paths-filter/releases)
- [Changelog](https://github.com/dorny/paths-filter/blob/master/CHANGELOG.md)
- [Commits](https://github.com/dorny/paths-filter/compare/de90cc6fb38fc0963ad72b210f1f284cd68cea36...9d7afb8d214ad99e78fbd4247752c4caed2b6e4c)

---
updated-dependencies:
- dependency-name: dorny/paths-filter
  dependency-version: 4.0.0
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

Spookerton committed:
tests workflow targets byond build 1679

Spookerton committed:
Merge pull request #35698 from HeyBanditoz/maybe_update_wiki_on_merge_fix

Fix "Notify Wiki On Merge" action failing on merge

Spookerton committed:
Merge pull request #35702 from Baystation12/dependabot/github_actions/dorny/paths-filter-4.0.0

Bump dorny/paths-filter from 3.0.2 to 4.0.0

Spookerton committed:
Merge pull request #35703 from Spookerton/spkrtn/bay/sys/test-1679

tests workflow targets byond build 1679

dependabot[bot] committed:
Bump dorny/paths-filter from 4.0.0 to 4.0.1

Bumps [dorny/paths-filter](https://github.com/dorny/paths-filter) from 4.0.0 to 4.0.1.
- [Release notes](https://github.com/dorny/paths-filter/releases)
- [Changelog](https://github.com/dorny/paths-filter/blob/master/CHANGELOG.md)
- [Commits](https://github.com/dorny/paths-filter/compare/9d7afb8d214ad99e78fbd4247752c4caed2b6e4c...fbd0ab8f3e69293af611ebaee6363fc25e6d187d)

---
updated-dependencies:
- dependency-name: dorny/paths-filter
  dependency-version: 4.0.1
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Builder-13 committed:
Translator implant change

HeyBanditoz committed:
Add food and recipes for ramen, ribs, and a BBQ sandwich

Thanks to EngagingLakeTahoe and thatdanguy23 for the sprites and code!

HeyBanditoz committed:
Admins can more easily modify jukebox tracks

HeyBanditoz committed:
Merge pull request #35707 from Builder-13/translate_implant

Translator implant change

HeyBanditoz committed:
Merge pull request #35708 from HeyBanditoz/bbq_ramen_foods

Add food and recipes for ramen, ribs, and a BBQ sandwich

HeyBanditoz committed:
Manual changelog update for #35707 and #35708

HeyBanditoz committed:
Merge pull request #35710 from HeyBanditoz/manual_changelog_update_0

Manual changelog update for #35707 and #35708

MuckerMayhem committed:
anomaly scanner can scan for unburied targets

sowelipililimute committed:
Allow nonhumans to insert cooking containers into appliances

sowelipililimute committed:
Streamline robot modules code & interactions

HeyBanditoz committed:
Merge pull request #35713 from sowelipililimute/work/jblackquill/let-nonhumans-cook

Allow nonhumans to insert cooking containers into appliances

MuckerMayhem committed:
Merge pull request #35680 from Spookerton/spkrtn/bay/cng/maptext-font

add maptext font preference

MuckerMayhem committed:
Merge pull request #35709 from HeyBanditoz/jukebox_admin_qol

Admins can more easily modify jukebox tracks

dependabot[bot] committed:
Bump actions/cache from 5.0.3 to 5.0.5

Bumps [actions/cache](https://github.com/actions/cache) from 5.0.3 to 5.0.5.
- [Release notes](https://github.com/actions/cache/releases)
- [Changelog](https://github.com/actions/cache/blob/main/RELEASES.md)
- [Commits](https://github.com/actions/cache/compare/cdf6c1fa76f9f475f3d7449005a359c84ca0f306...27d5ce7f107fe9357f9df03efb73ab90386fccae)

---
updated-dependencies:
- dependency-name: actions/cache
  dependency-version: 5.0.5
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

HeyBanditoz committed:
Fix the changelogs for real this time

Turns out there was a secret auth that actions/checkout
was introducing. Unintroduce it with some extra
insurance to boot.

Spookerton committed:
Merge pull request #35719 from HeyBanditoz/fix_the_changelogs_for_real_this_time

Fix the changelogs for real this time

Bay12Bot committed:
Automatic changelog generation for PR #35719 [ci skip]

HeyBanditoz committed:
Merge pull request #35714 from sowelipililimute/work/jblackquill/module-streamlining

Streamline robot modules code & interactions

Bay12Bot committed:
Automatic changelog generation for PR #35714 [ci skip]

HeyBanditoz committed:
Clarify user's emote key for freedom implant

HeyBanditoz committed:
Reorganize how server logs are stored

Now is organized as:
data/logs/2026/04/18/1300-abcdefgh/round-abcdefgh.log
... qdel-abcdefgh.log, runtime-abcdefgh.log

Filename for hour and minute (1300) is UTC

Podszywacz committed:
changes_oxyphoron_from_catalyst_to_substrate

SierraKomodo committed:
Merge pull request #35722 from Podszywacz/un-infinities_your_deuterium

Bay12Bot committed:
Automatic changelog generation for PR #35722 [ci skip]

SierraKomodo committed:
Merge pull request #35704 from Baystation12/dependabot/github_actions/dorny/paths-filter-4.0.1

SierraKomodo committed:
Merge pull request #35717 from Baystation12/dependabot/github_actions/actions/cache-5.0.5

SierraKomodo committed:
Merge pull request #35706 from MuckerMayhem/alden-surface-scan

Bay12Bot committed:
Automatic changelog generation for PR #35706 [ci skip]

HeyBanditoz committed:
Fix borg dreamchecker inspection fails

HeyBanditoz committed:
Switch tracy libs to paradise-tracy

It writes to disk instead of network, rtracy needed to replay

HeyBanditoz committed:
Merge pull request #35720 from HeyBanditoz/freedom_implant_emote_key

Clarify user's emote key for freedom implant

Podszywacz committed:
adds_low_temp_req_to_deuterium_reaction

MuckerMayhem committed:
fix alden mode toggle

MuckerMayhem committed:
Merge pull request #35725 from MuckerMayhem/alden-surface-scan

fix alden mode toggle

Bay12Bot committed:
Automatic changelog generation for PR #35725 [ci skip]

MuckerMayhem committed:
user -> usr

MuckerMayhem committed:
Merge pull request #35726 from MuckerMayhem/alden-surface-scan

user -> usr

SierraKomodo committed:
Merge pull request #35679 from HeyBanditoz/paradise_tracy

SierraKomodo committed:
Merge pull request #35723 from HeyBanditoz/the_linter

SierraKomodo committed:
Merge pull request #35724 from Podszywacz/stops_GAS_from_accidentally_shitting_bricks_of_deuterium

Bay12Bot committed:
Automatic changelog generation for PR #35724 [ci skip]

HeyBanditoz committed:
Borg fix

MuckerMayhem committed:
Merge pull request #35727 from HeyBanditoz/borg_fix

Borg fix

HeyBanditoz committed:
Fix the notify wiki job once and for all

We'll just use curl and jq!

HeyBanditoz committed:
Merge pull request #35728 from HeyBanditoz/fix_notify_wiki_on_merge_once_and_for_all

Fix the notify wiki job once and for all

HeyBanditoz committed:
pull_request event can't read secrets, I guess

Spookerton committed:
Merge pull request #35729 from HeyBanditoz/fix_notify_wiki_on_merge_once_and_for_all

pull_request event can't read secrets, I guess

Builder-13 committed:
MMI language and AI constuction fix

HeyBanditoz committed:
Add verb to rip out organs

If the wound is large enough and the limb is broken,
you can reach in and rip organs out!

MuckerMayhem committed:
add a radiation protection rig module

HeyBanditoz committed:
Add machinery_by_type alist to machinery subsystem

Currently, the Torch has ~16,000 machines, so an interation over that
entire list including a typecheck can get unwieldy if ever done in a
Process(), so we can just hold each machinery's type in an associative
list, and its parent type in its parent lists, and call upon that when
wanting all machines of a certain type. You should never have to access
SSmachines.machinery.

Catches all loops and some funky `locate()`s. Leaves a few uses in the
unit tests and some funky verb places that aren't used often anyway.
(Malf AI)

HeyBanditoz committed:
Merge pull request #35730 from Builder-13/ai_and_languages

Brain languages and AI core fix

Bay12Bot committed:
Automatic changelog generation for PR #35730 [ci skip]

Spookerton committed:
Merge pull request #35721 from HeyBanditoz/server_log_organization

Reorganize how server logs are stored

HeyBanditoz committed:
Log /world/Topic() with ISO8601 timestamp

MuckerMayhem committed:
Merge pull request #35732 from MuckerMayhem/radiation-module

add a radiation protection rig module

Bay12Bot committed:
Automatic changelog generation for PR #35732 [ci skip]

HeyBanditoz committed:
Adjust cigarettes

* Smoketime brought back to old values
* All cigarettes will have their contents properly transaferred to mob
  * Test to ensure this doesn't break
* Move cigarettes and masks from New() to Initialize()

Lentye committed:
Negative solar panel fix.dm

HeyBanditoz committed:
Merge pull request #35736 from Lentye/patch-1

Negative solar panel generation value fix.dm

MuckerMayhem committed:
supermatter shutdown feature

SierraKomodo committed:
Merge pull request #35734 from HeyBanditoz/better_topic_logging

Log /world/Topic() with ISO8601 timestamp

SierraKomodo committed:
Merge pull request #35735 from HeyBanditoz/cig_rebalancing

Adjust cigarettes

Bay12Bot committed:
Automatic changelog generation for PR #35735 [ci skip]

MuckerMayhem committed:
Merge pull request #35653 from MuckerMayhem/shutdown-test-with-h2-line

add supermatter shutdown feature

Bay12Bot committed:
Automatic changelog generation for PR #35653 [ci skip]

Spookerton committed:
Merge pull request #35733 from HeyBanditoz/machinery_by_type

Add machinery_by_type alist to machinery subsystem

Spookerton committed:
Merge pull request #35731 from HeyBanditoz/organ_rip_out

Add verb to rip out organs

Bay12Bot committed:
Automatic changelog generation for PR #35731 [ci skip]

MuckerMayhem committed:
add the clown cake

MuckerMayhem committed:
Merge pull request #35738 from MuckerMayhem/clown-cake

add the clown cake

Bay12Bot committed:
Automatic changelog generation for PR #35738 [ci skip]

dependabot[bot] committed:
Bump actions/checkout from 6.0.2 to 6.0.3

Bumps [actions/checkout](https://github.com/actions/checkout) from 6.0.2 to 6.0.3.
- [Release notes](https://github.com/actions/checkout/releases)
- [Commits](https://github.com/actions/checkout/compare/v6.0.2...v6.0.3)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-version: 6.0.3
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

HeyBanditoz committed:
Revert "Add verb to rip out organs"

MuckerMayhem committed:
Merge pull request #35740 from Baystation12/revert-35731-organ_rip_out

Revert "Add verb to rip out organs"

Bay12Bot committed:
Automatic changelog generation for PR #35740 [ci skip]

Spookerton committed:
Merge pull request #35739 from Baystation12/dependabot/github_actions/actions/checkout-6.0.3

Bump actions/checkout from 6.0.2 to 6.0.3

MuckerMayhem committed:
up dmg from shocks when wet, ignite if fire stacks

MuckerMayhem committed:
Merge pull request #35741 from MuckerMayhem/water-electrocution

up damage from shocks when wet, ignite if fire stacks

Bay12Bot committed:
Automatic changelog generation for PR #35741 [ci skip]

EngagingLakeTahoe committed:
add new food and wine items

Spookerton committed:
Merge pull request #35742 from EngagingLakeTahoe/FoodandWine

New Food and Wines

Bay12Bot committed:
Automatic changelog generation for PR #35742 [ci skip]

HeyBanditoz committed:
Clean up /mob/new_player references

They shouldn't fail to collect now

HeyBanditoz committed:
Shieldgen events use SSmachinery's list instead of snowflake GLOB

Follow up to #35733

MuckerMayhem committed:
fix sm shutdown final stage

MuckerMayhem committed:
Merge pull request #35745 from MuckerMayhem/fix-sm-shutdown

fix sm shutdown final stage

MuckerMayhem committed:
Merge pull request #35744 from HeyBanditoz/shield_generator_no_glob

Shieldgen events use SSmachinery's list instead of GLOB

MuckerMayhem committed:
Merge pull request #35743 from HeyBanditoz/new_player_del_fails

Clean up /mob/new_player references

HeyBanditoz committed:
Ending message in " or ' won't append period

Builder-13 committed:
Psi borers

dependabot[bot] committed:
Bump actions/checkout from 6.0.3 to 7.0.0

Bumps [actions/checkout](https://github.com/actions/checkout) from 6.0.3 to 7.0.0.
- [Release notes](https://github.com/actions/checkout/releases)
- [Commits](https://github.com/actions/checkout/compare/v6.0.3...v7)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-version: 7.0.0
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

HeyBanditoz committed:
Merge pull request #35747 from Builder-13/psi_borers

Borer changes

Bay12Bot committed:
Automatic changelog generation for PR #35747 [ci skip]

HeyBanditoz committed:
Fix harebell and affelerin plant typing

Fixes #35688, brings PRODUCE_SEED produced type and New() in line with
each plant type

Builder-13 committed:
Borer fix

HeyBanditoz committed:
Merge pull request #35750 from Builder-13/borer_fix

Borer fix

Bay12Bot committed:
Automatic changelog generation for PR #35750 [ci skip]

dependabot[bot] committed:
Bump actions/setup-python from 6.2.0 to 6.3.0

Bumps [actions/setup-python](https://github.com/actions/setup-python) from 6.2.0 to 6.3.0.
- [Release notes](https://github.com/actions/setup-python/releases)
- [Commits](https://github.com/actions/setup-python/compare/a309ff8b426b58ec0e2a45f0f869d46889d02405...ece7cb06caefa5fff74198d8649806c4678c61a1)

---
updated-dependencies:
- dependency-name: actions/setup-python
  dependency-version: 6.3.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump actions/cache/save from 5.0.5 to 6.0.0

Bumps [actions/cache/save](https://github.com/actions/cache) from 5.0.5 to 6.0.0.
- [Release notes](https://github.com/actions/cache/releases)
- [Changelog](https://github.com/actions/cache/blob/main/RELEASES.md)
- [Commits](https://github.com/actions/cache/compare/27d5ce7f107fe9357f9df03efb73ab90386fccae...2c8a9bd7457de244a408f35966fab2fb45fda9c8)

---
updated-dependencies:
- dependency-name: actions/cache/save
  dependency-version: 6.0.0
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

Spookerton committed:
Merge pull request #35746 from HeyBanditoz/punct_adjustment

Ending message in " or ' won't append period

Spookerton committed:
Merge pull request #35749 from HeyBanditoz/issue_35688_harebell_affelerin_plant_fix

Fix harebell and affelerin plant typing

Bay12Bot committed:
Automatic changelog generation for PR #35749 [ci skip]

Spookerton committed:
Merge pull request #35752 from Baystation12/dependabot/github_actions/actions/setup-python-6.3.0

Bump actions/setup-python from 6.2.0 to 6.3.0

Spookerton committed:
Merge pull request #35748 from Baystation12/dependabot/github_actions/actions/checkout-7.0.0

Bump actions/checkout from 6.0.3 to 7.0.0

dependabot[bot] committed:
Bump actions/cache from 5.0.5 to 6.1.0

Bumps [actions/cache](https://github.com/actions/cache) from 5.0.5 to 6.1.0.
- [Release notes](https://github.com/actions/cache/releases)
- [Changelog](https://github.com/actions/cache/blob/main/RELEASES.md)
- [Commits](https://github.com/actions/cache/compare/27d5ce7f107fe9357f9df03efb73ab90386fccae...55cc8345863c7cc4c66a329aec7e433d2d1c52a9)

---
updated-dependencies:
- dependency-name: actions/cache
  dependency-version: 6.0.0
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump actions/cache/restore from 5.0.5 to 6.1.0

Bumps [actions/cache/restore](https://github.com/actions/cache) from 5.0.5 to 6.1.0.
- [Release notes](https://github.com/actions/cache/releases)
- [Changelog](https://github.com/actions/cache/blob/main/RELEASES.md)
- [Commits](https://github.com/actions/cache/compare/27d5ce7f107fe9357f9df03efb73ab90386fccae...55cc8345863c7cc4c66a329aec7e433d2d1c52a9)

---
updated-dependencies:
- dependency-name: actions/cache/restore
  dependency-version: 6.0.0
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

Spookerton committed:
Merge pull request #35751 from Baystation12/dependabot/github_actions/actions/cache-6.0.0

Bump actions/cache from 5.0.5 to 6.1.0

Spookerton committed:
Merge pull request #35753 from Baystation12/dependabot/github_actions/actions/cache/restore-6.0.0

Bump actions/cache/restore from 5.0.5 to 6.1.0

Spookerton committed:
Merge pull request #35754 from Baystation12/dependabot/github_actions/actions/cache/save-6.0.0

Bump actions/cache/save from 5.0.5 to 6.0.0

HeyBanditoz committed:
Revert "Clean up /mob/new_player references"

This reverts commit f7ac2c596befaac3dd6280739bbf7c36cc8b686c.

MuckerMayhem committed:
Merge pull request #35755 from HeyBanditoz/revert_mob_qdel_changes

Revert "Clean up /mob/new_player references"

AttHawk committed:
lighter heating fix

SierraKomodo committed:
Merge pull request #35756 from LordNest/LN-zippo-AttHawk

[BUGFIX] Zippo object heating

Bay12Bot committed:
Automatic changelog generation for PR #35756 [ci skip]

HeyBanditoz committed:
Cleanup job age code

Fully removes the preferential character age weighting code for
heads-of-staff, it hasn't worked in years, and is supposed to runtime
(3 < list() doesn't runtime)

Corrects this behavior for offmap roles, although it was never used
because those don't have age requirements.

HeyBanditoz committed:
Some assorted fixes

Fixes exoplanet plants spawning broken, along with other various fixes!

MuckerMayhem committed:
Merge pull request #35759 from HeyBanditoz/job_age_cleanup

Cleanup job age code

MuckerMayhem committed:
Merge pull request #35760 from HeyBanditoz/assorted_fixes

Some assorted fixes
